backend/JavaWebService/src/main/java/com/kainos/ea/resources/CompetencyElement.java

```diff
@@ -0,0 +1,65 @@
+package com.kainos.ea.resources;
+
+public class CompetencyElement {
+    private int competencyElementID;
+    private String elementName;
+    private String elementDescription;
+    private int competencyID;
+    private int compBandID;
+    private String competencyName;
+    public CompetencyElement(int competencyElementID, int competencyID, String elementName,
+                             int compBandID, String elementDescription){
+        this.competencyElementID = competencyElementID;
+        this.competencyID = competencyID;
+        this.elementName = elementName;
+        this.compBandID = compBandID;
+        this.elementDescription = elementDescription;
+    }
+    public CompetencyElement(int competencyElementID, int competencyID, String elementName,
+                             int compBandID, String elementDescription, String competencyName){
+        this.competencyElementID = competencyElementID;
+        this.competencyID = competencyID;
+        this.elementName = elementName;
+        this.compBandID = compBandID;
+        this.elementDescription = elementDescription;
+        this.competencyName = competencyName;
+    }
+
+    public CompetencyElement(int competencyElementID, int compBandID, int competencyID, String elementName, String elementDescription){
+        this.competencyElementID = competencyElementID;
+        this.compBandID = compBandID;
+        this.competencyID = competencyID;
+        this.elementName = elementName;
+        this.elementDescription = elementDescription;
+    }
+    //for getting the competency names
+    public CompetencyElement(int competencyID, String competencyName){
+        this.competencyID = competencyID;
+        this.competencyName = competencyName;
+    }
+
+    public CompetencyElement(){}
```

Comment:
Are all 5 constructors necessary?

Also, remember what Jeremy said about using `this(...)`